modules/orca/main.tf

```diff
@@ -36,6 +36,7 @@ module "orca_lambdas" {
   orca_sqs_status_update_queue_id     = module.orca_sqs.orca_sqs_status_update_queue_id
   orca_sqs_status_update_queue_arn    = module.orca_sqs.orca_sqs_status_update_queue_arn
   restore_object_role_arn             = module.orca_iam.restore_object_role_arn
+  lambda_log_retention_in_days        = var.lambda_log_retention_in_days
```

Comment:
This is an optional variable since it has a default value. 

Place it under `## OPTIONAL` heading. Currently, it is under 
`## REQUIRED`